src/api/local-api.tsx

```diff
@@ -222,11 +224,14 @@ export class DatacoreLocalApi {
         return <LineSpanEmbed path={resolvedPath} start={start} end={end} />;
     }
 
+
     ///////////
     // Views //
     ///////////
 
+    public TaskList = TaskList;
     public VanillaTable = VanillaTable;
+		public Card = Card;
```

Comment:
I recommend running `yarn run format` for this. Also, it looks like your whitespace settings don't agree with the repo default (which is 4 spaces for tab) - you can edit that setting on a per-repo setting in most editors.